UnityProject/Assets/Scripts/Core/Editor/MiscFunctions_RRT.cs

```diff
@@ -134,10 +134,9 @@ public static void Reset()
 		public static void Generate()
 		{
 
-			bool True = true;
```

Comment:
lol

Reply:
yeah, first equal true The second one equal false